semgrep_output_v1.atd

```diff
@@ -1812,4 +1823,5 @@ type function_return
   | RetSarifFormat of sarif_format_return
   | RetContributions of contributions
   | RetFormatter of string
+  | RetValidate of validation_return
```

Comment:
Same here, just RetValidate of bool (* whether all the rules have valid syntax *)

Reply:
changed!